tsconfig.dev.json

```diff
@@ -8,7 +8,8 @@
     "inlineSources": true,
     "lib": [
       "dom",
-      "ES2019"
+      "ES2020",
+      "ES2020.Promise"
```

Comment:
What prompted this?

Reply:
Can probably switch back to be cleaner... `Promise.allSettled` was added with es2020. Didn't end up using it.